src/Data/NodeResolver.php

```diff
@@ -144,7 +158,7 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 			$pathinfo         = str_replace( '%', '%25', $pathinfo );
 
 			list( $req_uri ) = explode( '?', $pathinfo );
-			$home_path       = trim( wp_parse_url( home_url(), PHP_URL_PATH ), '/' );
+			$home_path       = trim( (string) wp_parse_url( home_url(), PHP_URL_PATH ), '/' );
```

Comment:
We know `home_url()` will give us a valid value, so we can coerce it instead of worrying about `string|false|null` like we need to [above](https://github.com/wp-graphql/wp-graphql/pull/2538/files?diff=split&w=1#r979216937)